src/components/Tables/Table.tsx

```diff
@@ -48,7 +48,8 @@ const dataToRows = (data: Datum[], headers: Headers): Row[] => {
         const row: Row = { rowId: currentIndex };
 
         for (const headerKey of Object.keys(headers)) {
-            row[headerKey] = datum[headerKey] ?? "";
+            const databaseValue = datum[headerKey] ?? "";
+            row[headerKey] = Array.isArray(databaseValue) ? databaseValue.join() : databaseValue;
```

Comment:
Is the new section arises from the database changes? Should this be in a separate PR if it does not relate to type checking? Just afraid that this might effect others that are using the Table component.

Reply:
It maybe should be in another PR, but it does directly relate to the type checking - without this, the type checker fails!
Previously it didn't check types 'that well' so when the database types changed it allowed it - but there is failing code on main! :eyes: